sdk/samples/iot/README.md

```diff
@@ -138,6 +138,7 @@ To run the samples, ensure you have the following programs or tools installed on
     ```
 
   - For non-Linux based systems, download the [OpenSSL 1.1.1 LTS](https://www.openssl.org/source/openssl-1.1.1g.tar.gz) command line utility and follow the downloaded INSTALL document.
```

Comment:
There are many issues in original implementation of this the C SDK you're cleaning up, which is great.  But one thing I'd like to see carried forward is the warning https://github.com/Azure/azure-iot-sdk-c/blob/master/tools/CACertificates/CACertificateOverview.md#warning, or some such after review with Azure SDK team.

This sample is needed at getting people going but basing a deployment off of keys generated on a dev's prototype box isn't something we should be encouraging.  I recommend you reach out to Eustace and Chrissie (I can't find them on GitHub) as IoT Security && DPS might have a better production quality guidance we should point folks to for productizing.

Reply:
The very clear explicit call out is nice.  I had boldened warnings in this readme, but your suggestion is MUCH more clear.

Reply:
I put it in the script itself with a `Write-Warning` so hopefully people pay attention to that.